src/models/StakeRequestRepository.ts

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+/* eslint-disable class-methods-use-this */
+
+import {
+  DataTypes, Model, InitOptions,
+} from 'sequelize';
+
+class StakeRequestModel extends Model {}
+
+export interface StakeRequestAttributes {
+  stakeRequestHash: string;
+  messageHash: string;
+  amount: number;
+  beneficiary: string;
+  gasPrice: number;
+  gasLimit: number;
+  nonce: number;
+  gateway: string;
+  stakerProxy: string;
+}
+
+export interface StakeRequest extends StakeRequestAttributes {
```

Comment:
It is difficult to understand the use of `StakeRequest` and `StakeRequestAttributes`
Documentation for these will be helpful.

Reply:
Ok.